cmake/compiler/gcc/target.cmake

```
@@ -34,7 +34,21 @@ if(NOT DEFINED NOSYSDEF_CFLAG)
   set(NOSYSDEF_CFLAG -undef)
 endif()
 
-foreach(file_name include/stddef.h include-fixed/limits.h)
+# GCC-13, does not install limits.h on include-fixed anymore
+# https://gcc.gnu.org/git/gitweb.cgi?p=gcc.git;h=be9dd80f933480
+# Add check for GCC version >= 13.1
+execute_process(
+    COMMAND ${CMAKE_C_COMPILER} -dumpversion
+    OUTPUT_VARIABLE TEMP_COMPILER_VERSION
```

Please use lower case for variables not intended to be used outside this file.
```suggestion
    OUTPUT_VARIABLE temp_compiler_version
```

a bit unfortunate that CMAKE_C_COMPILER_VERSION is not available at this point in time.

---

Done.